clang/include/clang/Sema/Sema.h

```diff
@@ -4101,8 +4101,8 @@ class Sema final : public SemaBase {
   /// (or member function template). The exception-specification was parsed
   /// after the method itself was declared.
   void actOnDelayedExceptionSpecification(
-      Decl *Method, ExceptionSpecificationType EST,
-      SourceRange SpecificationRange, ArrayRef<ParsedType> DynamicExceptions,
+      Decl *D, ExceptionSpecificationType EST, SourceRange SpecificationRange,
```

Comment:
Can you update the commento to explain that D can be the declaration of a friend function (template) ?

Reply:
@cor3ntin Updated